docker/Dockerfile

```diff
@@ -0,0 +1,38 @@
+ARG REMAGE_BASE_FLAVOR="latest"
+
+FROM gipert/remage-base:$REMAGE_BASE_FLAVOR
+
+LABEL maintainer.name="Luigi Pertoldi"
+LABEL maintainer.email="gipert@pm.me"
+
+ARG REMAGE_VERSION="v0.2.0"
```

Comment:
should we leave the fixed verison in here?

Reply:
It should be overridden by the build hook.